pkg/rc/farm.go

```diff
@@ -89,7 +88,7 @@ func NewFarm(
 // Set the metrics registry to use (see
 // https://godoc.org/github.com/rcrowley/go-metrics#Registry). If set,
 // performance metrics will be recorded to the registry.
-func (rcf *Farm) SetMetricsRegistry(registry metrics.Registry) error {
+func (rcf *Farm) SetMetricsRegistry(registry rcmetrics.MetricsRegistry) error {
```

Comment:
Maybe this is a broader question - why does anything in this file need to be aware of the underlying `rcmetrics.MetricsRegistry`? As far as I can tell, it's an implementation detail of `Metrics`, not of the RC farm. The logger appears to be passed in directly without alteration from the init of the farm to the init of `&rcmetrics.Metrics{}` on line 82.